bin/main.rb

```diff
@@ -1,2 +1,74 @@
 #!/usr/bin/env ruby
-puts 'Hello World!'
+# rubocop:disable Metrics/MethodLength
+def players_info
+  puts 'Welcome to Ruby\'s Tic Tac Toe!'
+  begin
+    puts 'Enter Player 1 name: '
+    player1 = gets.chomp.capitalize
+    puts ''
+    raise StandardError, player1 if player1.length < 3
+  rescue StandardError => e
+    puts "Invalid name \"#{e}\". The name should have at least 3 characters."
+    puts ''
+    retry
+  end
+
+  begin
+    puts 'Enter Player 2 name: '
+    player2 = gets.chomp.capitalize
+    puts ''
+    raise StandardError, player2 if player2.length < 3
+  rescue StandardError => e
+    puts "Invalid name \"#{e}\". The name should have at least 3 characters."
+    puts ''
+    retry
+  end
+
+  puts "#{player1} will play with X and #{player2} will play with O."
+  sleep 2
+  puts ''
+  puts 'Let\'s start!'
+  puts ''
+  sleep 2
+  puts `clear`
```

Comment:
- [ ] This and other similar code will cause the game to crash on the Windows environment because Windows do not know `clear` but `cls`. If you want to clear the screen use  : -
```
system 'cls'
system 'clear'
```
Which will work on Linux, Mac and Windows